src/Testing/TestResponseMixin.php

```diff
@@ -114,4 +117,101 @@ public function assertGraphQLErrorFree(): \Closure
             return $this;
         };
     }
+
+    public function assertGraphQLSubscriptionAuthorized(): \Closure
+    {
+        return function ($testClassInstance): TestResponse {
```

Comment:
type?

Reply:
Wouldn't adding type there make problems for php 7.4? That's why I did the assert not really sure though

Reply:
We require PHP 8 now.